libr/core/cbin.c

```diff
@@ -2048,6 +2053,9 @@ static int bin_symbols(RCore *r, int mode, ut64 laddr, int va, ut64 at, const ch
 		} else if (IS_MODE_SIMPLEST (mode)) {
 			const char *name = sn.demname? sn.demname: r_symbol_name;
 			r_cons_printf ("%s\n", name);
+		} else if (IS_MODE_RAD (mode) && is_special_symbol (symbol)) {
```

Comment:
Why chk twice just call is special inside the block below

Reply:
i did this to avoid a new level of indentation caused by the `if`, but... fixed it ;)